wiki/Guides/How_to_Use_the_Offset_Wizard/en.md

```diff
@@ -1,3 +1,25 @@
 # How to use the offset wizard
+*See also: [Universal Offset](/wiki/Universal_offset)*
 
-This guide was incorrect and has been removed.
+If the [Hitsounds](/wiki/Beatmapping/Hitsound) feel out of sync with the music, in every beatmap you play, you will more than likely need to change your [Universal Offset](/wiki/Universal_offset). The easiest way to do this is by using the **Offset Wizard**.
+
+Follow these steps to adjust your Universal Offset using the Offset Wizard:
+
+**Opening the Offset Wizard**:
+
+1. Open osu!.
+2. Go to the `Options` menu, or press `Ctrl` + `O`.
+3. Search "offset" in the quick-search and click on `Offset wizard`.
+
+The Offset Wizard will then open, playing the song that was previously playing in osu!, while a green bar scrolls across the screen and a metronome plays.
+
+**Using the Offset Wizard to calibrate your Universal Offset**:
+
+1. Listen carefully to the song and the metronome playing together.
+2. Start adjusting the Universal Offset up or down, using the `Up arrow` or `Down arrow` keys, or the `Scroll wheel`.
+   - If the hitsounds seem like they are playing too early in-game, you will need to *decrease* your Universal Offset.
+   - If the hitsounds seem like they are playing too late in-game, you will need to *increase* your Universal Offset.
+3. Adjust it until you hear the metronome line up perfectly with the song, using the scrolling bar on the screen to help you.
+4. Once you have successfully calibrated your Universal Offset, press the `Back` button at the bottom left to exit the Offset Wizard.
```

Comment:
```suggestion
4. Once you have successfully calibrated your universal offset, press the `Back` button at the bottom left to exit the Offset Wizard.
```

Reply:
Thanks, didnt know not to capitalize universal offset (im assuming because its not the topic of the article) fixed everything you suggested except one thing(which i have commented on) thanks again :)